index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My_Microverse_Portfolio_Project</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="toolbar">
+      <h1>Welcome</h1>
+      <button class="btn-trigger">
+        <img src="./assets/img/union.svg" alt="button"/>
+      </button>
+    </header>
```

Comment:
- [ ] Good idea using the  `<header>` tag to wrap the content of the header, However, let's use `<a>` tag for the Welcome text since it is considered as the logo, and a logo is supposed to be clickable to take you to the home page 😉 
- [ ] This was an excellent idea to have a button wrapping the hamburger icon 👏 , however, let's just keep the button inside of a `<nav>` tag since that's where all the menu links are supposed to be located and the hamburger represents the links on the mobile version, so keeping it inside the `<nav>` tag as well makes more sense 😉